core/src/pages/governance.astro

```diff
@@ -0,0 +1,315 @@
+---
+import Container from '../components/layout/Container.astro';
+import Divider from '../components/layout/Divider.astro';
+import PageHeader from '../components/layout/PageHeader.astro';
+import Layout from '../layouts/Layout.astro';
+
+import Quotation from '@/components/ui/Quotation.astro';
+import { getCollection, getEntry } from 'astro:content';
+import Citation from '../components/ui/Citation.astro';
+
+const teams = await getCollection('teams');
+const foundationBoard = await getEntry('teams', '010_foundation-board');
+const steeringCommittee = await getEntry('teams', '000_steering-committee');
+---
+
+<Layout title="Governance">
+  <PageHeader text="Governance" />
+  <Container class="py-8 leading-relaxed font-extralight">
+    <h2
+      class="font-heading text-secondary-afghani-blue mt-8 text-4xl font-bold"
+    >
+      Nix Leadership Bodies
+    </h2>
+    <p class="mb-6">The Nix community is guided by two leadership bodies:</p>
+    <div class="mt-2 mb-3 grid gap-4 md:grid-cols-2">
+      <div
+        class="bg-secondary-afghani-blue-95 flex w-full flex-col gap-2 rounded-xl p-4"
+      >
+        <h3
+          class="font-heading text-secondary-afghani-blue text-3xl leading-none font-bold"
+        >
+          The Steering Committee
+        </h3>
+        <span class="text-secondary-afghani-blue-35 text-lg">
+          7 elected individuals to represent the community
+        </span>
+        <span>
+          <b>Responsibilities:</b> Technical and community decisions and delegation
+        </span>
+        <span>
+          <b>Members:</b>
+          {
+            steeringCommittee.data.members
+              .map((member) => member.name)
```

Comment:
I think it would be better to use GitHub handles (with a link), because that's more recognisable